config/core/test-configs.yaml

```diff
@@ -2366,6 +2378,16 @@ test_configs:
       - baseline
       - baseline-nfs
 
+  - device_type: kontron-kswitch-d10-mmt-6g-2gs
+    test_plans:
+      - baseline
```

Comment:
OK https://staging.kernelci.org/test/plan/id/6296bed65785fb771ce12c16/

Reply:
OK with all buildroot results this time:
https://staging.kernelci.org/test/plan/id/62a14bc4d17c86804474f24a/